Back-End/controllers/students_controllers.js

```diff
@@ -0,0 +1,56 @@
+const db = require('../db');
+
+const categoryCourses_s = (req, res) => {
+ console.log("REQ Params: ", req.params.id)
+ let sql = `SELECT * FROM courses WHERE category_id = ${req.params.id}`
+ let query = db.query(sql, (err, result) => {
```

Comment:
- No need for the console log
- Why `let sql` and not `const`, also you could just rename it to query instead of sql
- remove `let query` since you are not using the variable anywhere

Reply:
Noted. `Console.log() `will be removed in future submittals before the pull request.
Noted, `sql `will be named query and the let to const. 
Noted, `let query` will be removed.